### PR TITLE
LSE-99 Move the Sign in button

### PR DESCRIPTION
**Pull request by RuslanSkiraRaccoongang**
**Description:** Move the Sign in button. Replace by the button with registration from other services

[**Youtrack:**](https://youtrack.raccoongang.com/issue/LSE-99)

**Reviewers:**
- [ ] @sendr

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

### CODE REVIEW

File: lms/djangoapps/student_account/context_processor.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    redirect_url=redirect_to,
+                ),
+                "registerUrl": pipeline.get_login_url(
+                    enabled.provider_id,
+                    pipeline.AUTH_ENTRY_REGISTER,
+                    redirect_url=redirect_to,
+                ),
+            }
+            context["providers" if not enabled.secondary else "secondaryProviders"].append(info)
+    return context
```
Comment (sendr) [inline lms/djangoapps/student_account/context_processor.py]: please, add an empty line

File: lms/djangoapps/student_account/context_processor.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# from common.djangoapps.third_party_auth.pipeline import get_provider_user_states
```
Comment (sendr) [inline lms/djangoapps/student_account/context_processor.py]: please remove this line if it is no longer needed, thanks

File: lms/djangoapps/student_account/context_processor.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+from third_party_auth import pipeline
+from .views import account_settings_context
+import third_party_auth
+from django.core.urlresolvers import reverse
+from student.helpers import get_next_url_for_login_page
```
Comment (sendr) [inline lms/djangoapps/student_account/context_processor.py]: reorder import please following by these rules https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#id12

File: lms/djangoapps/student_account/context_processor.py (hunk 1 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/student_account/context_processor.py]: added

File: lms/djangoapps/student_account/context_processor.py (hunk 2 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/student_account/context_processor.py]: remove

File: lms/djangoapps/student_account/context_processor.py (hunk 3 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/student_account/context_processor.py]: Done

**Pull request merged**